frontend/app/src/components/settings/accounting/rule/AccountingRuleSetting.vue

```diff
@@ -149,11 +152,17 @@ const showDeleteConfirmation = (item: AccountingRuleEntry) => {
     async () => await deleteAccountingRule(item)
   );
 };
+
+const getType = (eventType: string, eventSubtype: string) =>
+  getEventTypeData({
+    eventType,
+    eventSubtype
+  });
 </script>
 
 <template>
   <div>
-    <TablePageLayout :title="[t('accounting_settings.rule.title')]">
+    <TablePageLayout child :title="[t('accounting_settings.rule.title')]">
```

Comment:
Can we make the `TablePageLayout` here the root element and then move the form in it's default slot to remove the extra div?